app/views/catalog/_document_downloads.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if document_downloads.files.present? %>
```

Comment:
Sorry to be daft here.. where is this being rendered? I couldn't figure out how to view this.

Reply:
You're not daft, I didn't properly test for this.  Following https://github.com/projectblacklight/arclight/pull/808#discussion_r325833504, I realized what could be done to extend the View test suite.